week-2/Homework/mandatory/3-playing-computer.js

```diff
@@ -13,12 +13,12 @@
   Answer the following questions:
 
   1. This program throws an error. Why? (If you can't find it, try executing it).
-  2. Remove the line that throws the error.
-  3. What is printed to the console?
-  4. How many times is "f1" called?
-  5. How many times is "f2" called?
-  6. What value does the "a" parameter take in the first "f1" call?
-  7. What is the value of the "a" outer variable when "f1" is called for the first time?
+  2. Remove the line that throws the error. I removed line 37 because b dosn't have any value to print it.
+  3. What is printed to the console? x,a,d,e
```

Comment:
Almost right! What is requires is the values of x, a, d, e. See solution https://github.com/Ashaghel/js-solutions/blob/master/week-2/Homework/mandatory/3-playing-computer.js lines 19-23